test/jdk/security/infra/java/security/cert/CertPathValidator/certification/QuoVadisCA.java

```diff
@@ -57,10 +58,12 @@ public static void main(String[] args) throws Exception {
 
 class RootCA1G3 {
 
-    // Owner: CN=DigiCert QuoVadis TLS ICA QV Root CA 1 G3, O="DigiCert, Inc", C=US
+    // Owner: CN=DigiCert QuoVadis TLS ICA QV Root CA 1 G3, O="DigiCert,
+    // Inc", C=US
     // Issuer: CN=QuoVadis Root CA 1 G3, O=QuoVadis Limited, C=BM
     // Serial number: 2837d5c3c2b57294becf99afe8bbdcd1bb0b20f1
-    // Valid from: Wed Jan 06 12:50:51 PST 2021 until: Sat Jan 04 12:50:51 PST 2031
+    // Valid from: Wed Jan 06 12:50:51 PST 2021 until: Sat Jan 04 12:50:51
+    // PST 2031
```

Comment:
Can you put the "PST 2031" at the end of the previous line instead of on a new line? I think it would be more readable. Same comment applies to all the test certificates.